fe/fe-core/src/main/java/com/starrocks/server/NodeMgr.java

```diff
@@ -833,6 +832,15 @@ public void replayDropFrontend(Frontend frontend) {
         }
     }
 
+    public boolean hasSameHostFe(String host) {
+        for (Frontend fe : frontends.values()) {
+            if (fe.getHost().equals(host)) {
```

Comment:
Should we check for FQDN now that we support both IP & FQDN?

Reply:
done